modules/appeals_api/spec/support/shared_examples_for_appeal_status_updates.rb

```diff
@@ -96,13 +96,10 @@
     end
 
     context 'when veteran appellant without email provided' do
-      it 'gets the ICN and enqueues the appeal received job' do
+      it 'still enqueues the appeal received job, because the ICN can be used' do
         expect(AppealsApi::AppealReceivedJob.jobs.size).to eq 0
         instance_without_email.update_status(status: 'submitted')
         expect(AppealsApi::AppealReceivedJob.jobs.size).to eq 1
-
```

Comment:
Keeping a check about ICN here would now test the internals of the AppealReceivedJob, so I removed that part of this test case, but I kept it because I thought it was still valid to test that the job is at least triggered. The new specs for the AppealReceivedJob do include a test case for ICN.